applications/TrilinosApplication/custom_utilities/trilinos_assembling_utilities.h

```diff
@@ -0,0 +1,335 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//                   Vicente Mataix Ferrandiz
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "trilinos_space.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class TrilinosAssemblingUtilities
+ * @ingroup TrilinosApplication
+ * @brief The Trilinos assembling utilities
+ * @author Riccardo Rossi
+ * @author Vicente Mataix Ferrandiz
+ */
+template<class TSparseSpace>
```

Comment:
What is the point of templating on the SparseSpace? The functions here are trilinos-only